app/models/miq_worker/runner.rb

```diff
@@ -332,7 +332,10 @@ def do_work_loop
       begin
         heartbeat
         do_work
-      rescue TemporaryFailure
+      rescue TemporaryFailure => error
+        msg = "#{log_prefix} Temporary failure (message: '#{error.to_s}')"\
```

Comment:
as miq-bot noticed, no need for `to_s`

@jrafanie wdyt about the logging at this point?
And should we also log in the event_catcher to the provider specific logfile?